From f5683275eacd2d0e59414041ce4c77bb6a02bb69 Mon Sep 17 00:00:00 2001
From: Christian Stewart <christian@paral.in>
Date: Mon, 31 Aug 2020 16:40:50 -0700
Subject: [PATCH] mmc: dw_mmc: fix the spamming log message

Remove a log message which is written on the Odroid several times each second,
when it attempts to probe for a hot-plugged emmc card.

Signed-off-by: Christian Stewart <christian@paral.in>
---
 drivers/mmc/host/dw_mmc.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/mmc/host/dw_mmc.c b/drivers/mmc/host/dw_mmc.c
index 32001d43e453..2487bd851464 100644
--- a/drivers/mmc/host/dw_mmc.c
+++ b/drivers/mmc/host/dw_mmc.c
@@ -1274,13 +1274,6 @@ static void dw_mci_setup_bus(struct dw_mci_slot *slot, bool force_clkinit)
 		if ((clock != slot->__clk_old &&
 			!test_bit(DW_MMC_CARD_NEEDS_POLL, &slot->flags)) ||
 			force_clkinit) {
-			/* Silent the verbose log if calling from PM context */
-			if (!force_clkinit)
-				dev_info(&slot->mmc->class_dev,
-					 "Bus speed (slot %d) = %dHz (slot req %dHz, actual %dHZ div = %d)\n",
-					 slot->id, host->bus_hz, clock,
-					 div ? ((host->bus_hz / div) >> 1) :
-					 host->bus_hz, div);
 
 			/*
 			 * If card is polling, display the message only
-- 
2.28.0

